src/Http/Http.Abstractions/src/ConnectionInfo.cs

```diff
@@ -56,4 +59,24 @@ public abstract class ConnectionInfo
     public virtual void RequestClose()
     {
     }
+
+    /// <inheritdoc />
+    public override string ToString()
```

Comment:
Until we have a scenario where we'd use the result of `ToString`, I'd prefer putting this in an internal `DebuggerToString()` and have the display attribute call the method.

Reply:
+1 to not introducing public implementations of the ToString APIs until absolutely needed.